src/app/grid/grid-paste/grid-paste-sample.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="grid__wrapper">
     <div class="grid__menu">
-        <button igxButton="raised" [igxToggleAction]="dropdown" [igxDropDownItemNavigation]="dropdown">Paste Options</button>
+        <button igxButton="raised" [batchEditing]="true" [igxToggleAction]="dropdown" [igxDropDownItemNavigation]="dropdown">Paste Options</button>
```

Comment:
`button` with `bathcEditing` - priceless 🤣 
please move this to the grid bellow